_posts/2018-04-09-ce-qui-se-passe-dans-la-tete-d-un-developpeur.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Ce qui se passe dans la tête d'un développeur
+authors: philippe.blayo
+---
+
+Un soir, Hela poste un message d'erreur sur Slack :
+
+> Le site est cassé, je peux plus rentrer mes cas types :/
+
+![Une erreur est survenue](/img/posts/2018-04-02-ce-qui-se-passe-dans-la-tete-d-un-developpeur/la_fameuse_erreur.png)
+
+Hela en ce moment elle fait naître des incubateurs, dont
+celui des affaires sociales qu'elle accompagne personnellement.
+Elle a passé l'après-midi avec le groupe de travail
+de lutte contre le non recours aux droits.
+Elle simulait les conséquences de cette réforme quand l'erreur
+s'est produite.
+
+C'est important de simuler les conséquences d'une réforme avant de la voter.
+Parce qu'après c'est beaucoup plus compliqué à changer, et pendant
+ce temps des gens en subissent les conséquences.
+C'est un peu l'équivalent en programmation d'écrire des tests avant
+d'écrire le code.
+
+Je comprends qu'elle a des projections à rendre pour le lendemain.
+
+Et puis le site "cassé" est visité dix mille fois par jour, davantage
+quand un article de presse ou une émission le mentionne donc
+ça mérite mon attention.
```

Comment:
Il manque peut-être un élément de contexte, "je ne travaille pas sur ce sujet mais j'interprète son message comme une demande d'aide".

Ce que je trouve intéressant dans l'article c'est que derrière son côté très factuel il met en évidence certaines de nos heuristiques, par exemple "You see it, you own it" - mais sans à aucun moment les théoriser. Je pense qu'il faut garder ça, mais à la relecture il faut se projeter dans la tête des lecteurs et lectrices: quels sont les éléments de contexte dont tu disposes qui leur manquent pour donner du sens à la narration ? On n'a pas besoin de *tout* comprendre, mais en tant qu'auteur c'est à toi de réfléchir aux conclusions et inférences qui ont le plus d'importance pour toi et de donner les éléments de contexte qui permettent ces inférences.

Reply:
@Morendil : Merci pour ta suggestion. Je l'ai résumé de la manière suivante et suis intéressé de savoir si tu penses qu'il manque encore du contexte :

> Je ne travaille pas sur ce sujet mais j'interprète son message comme 
une demande d'aide. "You see it, you own it" : si je vois un problème, 
je tente de le corriger, même si d'autres sont plus légitimes ou plus 
compétents pour le faire.